
gaussian_blur_size: 9
gaussian_blur_sigma: 1
canny_edge_th: 50
hough_accum_resolution: 1
min_circle_dist: 40
hough_accum_th: 120
min_radius: 20
max_radius: 400
